core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -272,6 +273,8 @@ object TopicCommand extends Logging {
       val allConfigs = adminClient.describeConfigs(topics.map(new ConfigResource(Type.TOPIC, _)).asJavaCollection).values()
       val liveBrokers = adminClient.describeCluster().nodes().get().asScala.map(_.id())
       val topicDescriptions = adminClient.describeTopics(topics.asJavaCollection).all().get().values().asScala
+      val topicPartitions = topicDescriptions.flatMap(pi => pi.partitions().asScala.map(tpi => new TopicPartition(pi.name(), tpi.partition())))
+      val reassignments = adminClient.listPartitionReassignments(topicPartitions.toSet.asJava).reassignments().get()
```

Comment:
I'm not sure we have a nice way to handle this at the moment, but this API may not exist on older clusters. I guess we should try to catch `UnsupportedVersionException` and revert to the prior logic?

Reply:
Good point, adding it.